recurring_contract/migrations/14.0.1.0.3/pre-migration.py

```diff
@@ -1,6 +0,0 @@
-
```

Comment:
Why did you remove this migration? I don't find anything replacing it.

Reply:
I moved it in compassion-module :) it was a migration on a field defined in the sponsorship not in the recurring_contract